forte/data/data_store.py

```diff
@@ -307,23 +362,29 @@ def add_group_raw(
     def set_attribute(self, tid: int, attr_name: str, attr_value: Any):
         r"""This function locates the entry data with ``tid`` and sets its
         ``attr_name`` with `attr_value`. It first finds ``attr_id``  according
-        to ``attr_name``. `tid`, `attr_id`, and ``attr_value`` are
+        to ``attr_name``. ``tid``, ``attr_id``, and ``attr_value`` are
         passed to `set_attr()`.
+        Raises KeyError when ``tid`` or ``attr_name`` is not found.
```

Comment:
Use the correct `raise` index.

Reply:
Did that in line 433. Forget to delete this line